src/main/java/juja/sqlcmd/view/Console.java

```diff
@@ -0,0 +1,42 @@
+package juja.sqlcmd.view;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class Console implements View {
+    private InputStream inputStream;
+    private OutputStream outputStream;
+
+    public Console() {
+        this(System.out, System.in);
+    }
+
+    public Console(OutputStream outputStream, InputStream inputStream) {
```

Comment:
Maybe change order in signature (in,out)?
Now (in, out) in declaration, but (out, in) in constructors

Reply:
Yup, agree, that order is more logic